### ISSUE TITLE
1.7.0 still not working on java 11

### ISSUE THREAD

**Issue by davidkarlsen** [gaul/modernizer-maven-plugin#81]
```
[DEBUG] Populating class realm plugin>org.gaul:modernizer-maven-plugin:1.7.0
[DEBUG]   Included: org.gaul:modernizer-maven-plugin:jar:1.7.0
[DEBUG]   Included: org.codehaus.plexus:plexus-utils:jar:1.5.1
[DEBUG]   Included: junit:junit:jar:3.8.1
[DEBUG]   Included: org.ow2.asm:asm:jar:7.0
[DEBUG]   Included: org.ow2.asm:asm-commons:jar:7.0
[DEBUG]   Included: org.ow2.asm:asm-tree:jar:7.0
[DEBUG]   Included: org.ow2.asm:asm-analysis:jar:7.0
[DEBUG] Configuring mojo org.gaul:modernizer-maven-plugin:1.7.0:modernizer from plugin realm ClassRealm[plugin>org.gaul:modernizer-maven-plugin:1.7.0, parent: jdk.inte
rnal.loader.ClassLoaders$AppClassLoader@799f7e29]
[DEBUG] Configuring mojo 'org.gaul:modernizer-maven-plugin:1.7.0:modernizer' with basic configurator -->
[DEBUG]   (f) failOnViolations = true
[DEBUG]   (f) includeTestClasses = true
[DEBUG]   (f) javaVersion = 11
[DEBUG]   (f) outputDirectory = /Users/et2448/projects/tac/jfr/tac-jfr-server/jfr-srv-schemas/target/classes
[DEBUG]   (f) project = MavenProject: com.edb.fs.tac.jfr.srv:jfr-srv-schemas:7.0.0-SNAPSHOT @ /Users/et2448/projects/tac/jfr/tac-jfr-server/jfr-srv-schemas/pom.xml
[DEBUG]   (f) skip = false
[DEBUG]   (f) sourceDirectory = /Users/et2448/projects/tac/jfr/tac-jfr-server/jfr-srv-schemas/src/main/java
[DEBUG]   (f) testOutputDirectory = /Users/et2448/projects/tac/jfr/tac-jfr-server/jfr-srv-schemas/target/test-classes
[DEBUG]   (f) testSourceDirectory = /Users/et2448/projects/tac/jfr/tac-jfr-server/jfr-srv-schemas/src/test/java
[DEBUG]   (f) violationLogLevel = error
[DEBUG]   (f) violationsFiles = []
[DEBUG] -- end configuration --
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  24.976 s (Wall Clock)
[INFO] Finished at: 2018-11-10T10:47:51+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.gaul:modernizer-maven-plugin:1.7.0:modernizer (modernizer) on project jfr-srv-schemas: Execution modernizer of goal org.gaul:moderni
zer-maven-plugin:1.7.0:modernizer failed: This feature requires ASM7 -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.gaul:modernizer-maven-plugin:1.7.0:modernizer (modernizer) on project jfr-srv-schema
s: Execution modernizer of goal org.gaul:modernizer-maven-plugin:1.7.0:modernizer failed: This feature requires ASM7
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.builder.multithreaded.MultiThreadedBuilder$1.call (MultiThreadedBuilder.java:200)
    at org.apache.maven.lifecycle.internal.builder.multithreaded.MultiThreadedBuilder$1.call (MultiThreadedBuilder.java:196)
    at java.util.concurrent.FutureTask.run (FutureTask.java:264)
    at java.util.concurrent.Executors$RunnableAdapter.call (Executors.java:515)
    at java.util.concurrent.FutureTask.run (FutureTask.java:264)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1128)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:628)
    at java.lang.Thread.run (Thread.java:834)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution modernizer of goal org.gaul:modernizer-maven-plugin:1.7.0:modernizer failed: This feature requires ASM7
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:148)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
```

**Comment by gaul** [comment]
You might try enabling additional logging -- it looks like part of the stack trace got eaten somewhere.  Your log clearly shows asm 7.0 used but try running `mvn dependency:tree` to double-check?  I successfully ran OpenJDK 11 with modernizer 1.7.0 so I am not sure what specific code triggers this failure.  modernizer is at the mercy of the asm dependency so you might want to follow up with their issue tracker: https://gitlab.ow2.org/asm/asm/issues

**Comment by kincajou** [comment]
I have same error:
```
[ERROR] Failed to execute goal org.gaul:modernizer-maven-plugin:1.7.0:modernizer (modernizer) on project commons: Execution modernizer of goal org.gaul:modernizer-maven-plugin:1.7.0:modernizer failed: This feature requires ASM7 -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.gaul:modernizer-maven-plugin:1.7.0:modernizer (modernizer) on project commons: Execution modernizer of goal org.gaul:modernizer-maven-plugin:1.7.0:modernizer failed: This feature requires ASM7
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
	at org.codehaus.classworlds.Launcher.main(Launcher.java:47)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution modernizer of goal org.gaul:modernizer-maven-plugin:1.7.0:modernizer failed: This feature requires ASM7
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:145)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	... 21 more
Caused by: java.lang.UnsupportedOperationException: This feature requires ASM7
	at org.objectweb.asm.ClassVisitor.visitNestHost(ClassVisitor.java:150)
	at org.objectweb.asm.ClassReader.accept(ClassReader.java:541)
	at org.objectweb.asm.ClassReader.accept(ClassReader.java:391)
	at org.gaul.modernizer_maven_plugin.Modernizer.check(Modernizer.java:79)
	at org.gaul.modernizer_maven_plugin.Modernizer.check(Modernizer.java:84)
	at org.gaul.modernizer_maven_plugin.ModernizerMojo.recurseFiles(ModernizerMojo.java:311)
	at org.gaul.modernizer_maven_plugin.ModernizerMojo.recurseFiles(ModernizerMojo.java:304)
	at org.gaul.modernizer_maven_plugin.ModernizerMojo.recurseFiles(ModernizerMojo.java:304)
	at org.gaul.modernizer_maven_plugin.ModernizerMojo.recurseFiles(ModernizerMojo.java:304)
	at org.gaul.modernizer_maven_plugin.ModernizerMojo.recurseFiles(ModernizerMojo.java:304)
	at org.gaul.modernizer_maven_plugin.ModernizerMojo.recurseFiles(ModernizerMojo.java:304)
	at org.gaul.modernizer_maven_plugin.ModernizerMojo.recurseFiles(ModernizerMojo.java:304)
	at org.gaul.modernizer_maven_plugin.ModernizerMojo.execute(ModernizerMojo.java:220)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	... 22 more
```
My guess this is caused by using ASM7 feature while configuring for ASM5 version here:

https://github.com/gaul/modernizer-maven-plugin/blob/modernizer-maven-plugin-1.7.0/src/main/java/org/gaul/modernizer_maven_plugin/Modernizer.java#L139

**Comment by gaul** [comment]
Could you test this locally and submit a pull request if it fixes theissue?  Just `mvn install` and change the version used in `pom.xml` to 1.8.0-SNAPSHOT.

**Comment by gaul** [comment]
Fixed by #82.  I am traveling and my maven credentials are not on this computer so I will push a new release when I return in a week.

**Comment by gaul** [comment]
Pushed 1.7.1.
